opentelemetry-etw-logs/src/logs/reentrant_logprocessor.rs

```diff
@@ -0,0 +1,110 @@
+use std::fmt::Debug;
+
+use opentelemetry::logs::LogResult;
+use opentelemetry_sdk::export::logs::LogData;
+
+#[cfg(feature = "logs_level_enabled")]
+use opentelemetry_sdk::export::logs::LogExporter;
+
+use crate::logs::exporter::ExporterConfig;
+use crate::logs::exporter::*;
+
+/// This export processor exports without synchronization.
+/// This is currently only used in users_event exporter, where we know
+/// that the underlying exporter is safe under concurrent calls
+
+#[derive(Debug)]
+pub struct ReentrantLogProcessor {
+    event_exporter: ETWExporter,
+}
+
+impl ReentrantLogProcessor {
+    /// constructor
+    pub fn new(
+        provider_name: &str,
+        event_name: String,
+        provider_group: ProviderGroup,
+        exporter_config: ExporterConfig,
+    ) -> Self {
+        let exporter = ETWExporter::new(provider_name, event_name, provider_group, exporter_config);
+        ReentrantLogProcessor {
+            event_exporter: exporter,
+        }
+    }
+}
+
+impl opentelemetry_sdk::logs::LogProcessor for ReentrantLogProcessor {
```

Comment:
Just a thought, not directly related to this PR, but looking ahead, we can explore the possibility of combining the user_events and ETW log exporters, in line with the discussion for metrics in #46. It could be a nice way to streamline our code and cut down on redundant code like here.

Reply:
Yes! I thought on this originally. That should not be hard to accomplish.